io.github.lsfg.lsfg-vk.yml

```diff
@@ -0,0 +1,30 @@
+id: io.github.lsfg.lsfg-vk
+branch: '24.08'
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm20
+
+build-options:
+  prefix: /usr/lib/extensions/vulkan/lsfgvk
```

Comment:
if this is a vulkanlayer extension use `org.freedesktop.Platform.VulkanLayer.*` as the ID prefix.

Reply:
it's written specifically in the requirements https://docs.flathub.org/docs/for-app-authors/requirements#extensions-and-baseapps

> [Flatpak extensions](https://docs.flatpak.org/en/latest/extension.html) must prefix their ID with the extension point ID, which is either defined in one of the [runtimes](https://docs.flathub.org/docs/for-app-authors/runtimes) or in an app. If the extension point is defined in an app, it must use the app's ID as a prefix. They are exempt from the [domain control rules](https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain-or-repository).